GameData/RealismOverhaul/TestFlight_Generic_Engines.cfg

```diff
@@ -1,3 +1,55 @@
+// Pre-process: EngineConfigs/ put TESTFLIGHT nodes inside MEC/CONFIGs;
+// this lets us copy some fields from one to the other, but then the
+// FOR:zTestFlight processing steps need them directly in the PART node.
+// Must use BEFORE to preserve TestLite compatibility.
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]]:BEFORE[zTestFlight]
+{
+	@MODULE[ModuleEngineConfigs]
+	{
+		@CONFIG:HAS[@TESTFLIGHT:HAS[#ratedBurnTime]] { &ratedBurnTime = #$TESTFLIGHT/ratedBurnTime$ }
+		@CONFIG:HAS[@TESTFLIGHT:HAS[#ratedContinuousBurnTime]] { &ratedContinuousBurnTime = #$TESTFLIGHT/ratedContinuousBurnTime$ }
+		@CONFIG:HAS[@TESTFLIGHT:HAS[#testedBurnTime]] {
+			// The game doesn't show testedBurnTime anywhere!
+			// I like having it in the description for lack of something better
+			@description = #$description$ testedBurnTime=$TESTFLIGHT/testedBurnTime$
+		}
+
+		@CONFIG:HAS[@TESTFLIGHT]
+		{
+			|_ = CONFIG_wip // for the MM_PATCH_LOOP below
+			@TESTFLIGHT,* {
+				name = #$../name$
```

Comment:
Should be `%name = `?

Reply:
probably a good idea. the intent was to never provide a name in the TESTFLIGHT nodes themselves, but there's a pretty good chance I missed deleting some.

Reply:
This doesn't appear to have actually been done? I agree that it should be just to safeguard against any poorly written configs.